PowerEditor/src/Notepad_plus.rc

```diff
@@ -673,6 +673,8 @@ BEGIN
             MENUITEM "Show Non-Printing Characters",    IDM_VIEW_NPC
             MENUITEM "Show All Characters",             IDM_VIEW_ALL_CHARACTERS
             MENUITEM SEPARATOR
+            MENUITEM "Show C0, C1 Control and Unicode EOL Characters", IDM_VIEW_NPC_CCUNIEOL
+            MENUITEM SEPARATOR
             MENUITEM "Show Indent Guide",               IDM_VIEW_INDENT_GUIDE
             MENUITEM "Show Wrap Symbol",                IDM_VIEW_WRAP_SYMBOL
         END

```

Comment:
Please do the following instead:
```
        BEGIN
            MENUITEM "Show Space and Tab",                     IDM_VIEW_TAB_SPACE
            MENUITEM "Show End of Line",                       IDM_VIEW_EOL
            MENUITEM "Show Non-Printing Characters",           IDM_VIEW_NPC
            MENUITEM "Show Control Characters && Unicode EOL", IDM_VIEW_NPC_CCUNIEOL
            MENUITEM "Show All Characters",                    IDM_VIEW_ALL_CHARACTERS
            MENUITEM SEPARATOR
            MENUITEM "Show Indent Guide",                      IDM_VIEW_INDENT_GUIDE
            MENUITEM "Show Wrap Symbol",                       IDM_VIEW_WRAP_SYMBOL
        END
```

Reply:
@donho 
currently it is not part of "Show All Characters" won't it causes confusion if it is next to "Show All Characters" and not separated. Or should I include it in "Show All Characters".

Reply:
@ozone10 
If  "Show All Characters" shows NPC, I don't see why it doesn't show Control Characters & Unicode EOL.
So please include c0, c1 & Unicode EOL in "Show All Characters".